src/Cli/dotnet/commands/dotnet-run/RunCommand.cs

```diff
@@ -329,26 +334,21 @@ static ICommand CreateCommandFromRunProperties(ProjectInstance project, RunPrope
                 return command;
             }
 
-            static void InvokeRunArgumentsTarget(ProjectInstance project, string[] restoreArgs, VerbosityOptions? verbosity)
+            static void InvokeRunArgumentsTarget(ProjectInstance project, string[] restoreArgs, BinlogArgs? binlogArgs, VerbosityOptions? verbosity)
```

Comment:
@mariam-abdulla here's how I used the bl option to correctly forward along binlog information. Add the option to your parser, get the value from the parser, and then use the data structure to create the `BinaryLogger` you need (if any).